tutorials/Anatomy of a Device.ipynb

```diff
@@ -25,8 +25,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Runs EPICS IOC(s) with simulated hardware in leiu of actual motors, detectors.\n",
-    "!supervisor/start_supervisor.sh"
+    "# Monitor EPICS IOC(s) with simulated hardware in leiu of actual motors, detectors.\n",
```

Comment:
The word "monitor" is an overloaded term in this context. "Monitoring an IOC" might be confused with `camonitor`. I think we should be a little more specific about why this line is here and what the student is supposed to do. Maybe:

> Below, we will connect to EPICS IOC(s) controlling simulated hardware in leiu of actual motors, detectors. The IOCs should already be running in the background. Run this command to verify that they are running: it should produce output with `RUNNING` on each line. In the event of a problem, edit this command to replace `status` with `restart all` and run again.